topics/slack/index.md

```diff
@@ -0,0 +1,13 @@
+---
+aliases: slack-messaging, slack-chat
```

Comment:
The `slack-messaging` topic is empty, and `slack-chat` has only two repositories that look unrelated. Let's just get rid of the alias.

```suggestion
```